website/docs/cdktf/debugging.mdx

```diff
@@ -0,0 +1,59 @@
+---
+page_title: Debugging
+description: >-
+  Learn how to troubleshoot problems in your CDKTF application.
+---
+
+# Debugging
+
+## Increasing the verbosity
+
+To get more information about what CDKTF is doing and what problems have occured by setting the `CDKTF_LOG_LEVEL` environment variable to `DEBUG`. If you want to share the logs you can set `CDKTF_LOG_FILE_DIRECTORY` to a file path, a `cdktf.log` file with all logs will be written.
```

Comment:
The `## CDKTF_LOG_LEVEL` section in the environment variable docs case it as `debug`, we should use the lower case version everywhere.

Reply:
(resolved by one of your commits)